backend/test_firestore.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
```

Comment:
can we use `logrus` here for consistency? it's the log package used in `backend/main.go`